src/esp/gfx/Simulator.h

```diff
@@ -88,7 +88,7 @@ class Simulator {
   int getPhysicsObjectLibrarySize();
 
   // remove object objectID instance in sceneID
-  void removeObject(const int objectID, const int sceneID = 0);
+  int removeObject(const int objectID, const int sceneID = 0);
```

Comment:
If I get it correctly, [my comment](https://github.com/facebookresearch/habitat-sim/pull/213#discussion_r325322324) from #213 on the return type change should have been on this PR instead? :)

Reply:
The comments should be up to date now here.